### ISSUE TITLE
Does dochead work with Blaze in 1.2

### ISSUE THREAD

**Issue by txs** [kadirahq/meteor-dochead#33]
The dochead doesn't work for me in 1.2 with blaze.

In the head code, I set the default to run in flowrouter trigger enter and in router action.
The head show the code 2 times.
But in facebook debug, I just doesn't make it work!

**Comment by maxenceC** [comment]
Exact same thing here, blaze@2.1.7 : 
- I use DocHead in flowrouter trigger enter, as well as in a Meteor.startup()
- If I open google inspector, meta tags are there
- Spiderable is installed
- In facebook debug and google webmaster tools , There are no meta tags

**Comment by krishaamer** [comment]
Has this been resolved?

**Comment by txs** [comment]
Mine even  with Google  is not there.  I think I have a issue  with phantom
in mup

**Comment by Pagebakers** [comment]
This will make it work with Blaze

``` javascript
FlowRouter.route('/post/:postId', {
  name: 'post',
  action(params) {
    if (Meteor.isClient) {
      BlazeLayout.render('layout')
    } else {
      Meteor.subscribe('posts', params.postId)
      if ((post = Posts.findOne(params.postId))) {
        DocHead.addMeta({
          property: 'og:image',
          content: post.image
        })
      }
    }
  }
})
```

**Comment by txs** [comment]
no!!! I rewrite it all with react  router! Thanks  BTW  I'll try it
